tests/monitoring/monitoring_service/test_handlers.py

```diff
@@ -186,6 +186,30 @@ def test_channel_closed_event_handler_closes_existing_channel(
     assert_channel_state(context, ChannelState.CLOSED)
 
 
+def test_channel_closed_event_handler_idempotency(
```

Comment:
Maybe we can find a good way to test idempotency for all events without writing a separate test for each event in the future.